test/WebJobs.Script.Tests/Configuration/ApplicationInsightsConfigurationTests.cs

```diff
@@ -24,7 +24,7 @@ public void ServicesDisabled_InPlaceholderMode()
                 host = new HostBuilder()
                     .ConfigureLogging((context, builder) =>
                     {
-                        ScriptHostBuilderExtensions.ConfigureApplicationInsights(context, builder);
+                        ScriptHostBuilderExtensions.ConfigureApplicationInsights(builder, context);
```

Comment:
This is an extension method right?

```suggestion
                        builder.ConfigureApplicationInsights(context);
```